tools/tu_collector/tu_collector/tu_collector.py

```diff
@@ -371,13 +375,28 @@ def main():
                           help="Use an already existing JSON compilation "
                                "command database file specified at this path.")
 
-    parser.add_argument('-z', '--zip', dest='zip', type=str, required=True,
-                        help="Output ZIP file.")
     parser.add_argument('-f', '--filter', dest='filter',
                         type=str, required=False,
-                        help="This flag restricts the collection on the build "
-                             "actions of which the compiled source file "
-                             "matches this path. E.g.: /path/to/*/files")
+                        help="If '--zip' option is given this flag restricts "
+                             "the collection on the build actions of which "
+                             "the compiled source file matches this path. "
+                             "If '--dependencies' option is given this flag "
+                             "specify a header file to get source file "
+                             "dependencies for. E.g.: /path/to/*/files")
+
+    output_args = parser.add_argument_group(
+        "output arguments",
+        "Specify the output type.")
+
+    output_args = output_args.add_mutually_exclusive_group(required=True)
+    output_args.add_argument('-z', '--zip', dest='zip', type=str,
+                             help="Output ZIP file.")
+
+    output_args.add_argument('-d', '--dependencies', dest='dependencies',
```

Comment:
It could be `--dependents` instead. It would be even better if somehow we could include that the result doesn't contain header files, even if those are dependents as well. Or at lease we could write it in the help message of this flag.